docs/team/matthewphua.md

```diff
@@ -10,6 +10,38 @@ Command Line Interface (CLI).
 **Code contributed:**
 [Tp Dashboard contributions](https://nus-cs2113-ay2223s1.github.io/tp-dashboard/?search=&sort=groupTitle&sortWithin=title&timeframe=commit&mergegroup=&groupSelect=groupByRepos&breakdown=true&checkedFileTypes=docs~functional-code~test-code~other&since=2022-09-16&tabOpen=true&tabType=authorship&tabAuthor=matthewphua&tabRepo=AY2223S1-CS2113-T18-1b%2Ftp%5Bmaster%5D&authorshipIsMergeGroup=false&authorshipFileTypes=docs~functional-code~test-code&authorshipIsBinaryFileTypeChecked=false&authorshipIsIgnoredFilesChecked=false)
 
+* **New Feature**: Abstract Command Class Architecture
+  * What it does: This feature serves as the foundation for all commands within our CLI product.
+  * Justification: All commands need an overall similar structure to provide continuity within implementation.
+  * Highlights: Had to make an overall structure for all command actions.
+  Implemented the OOP model into our design. 
+* **New Feature**: AddCommand
+  * What it does: Adds the review to storage.
+  * Justification: This is the most essential behavior to our application. 
+  It allows the user to add own personal reviews to storage.
+  * Highlights: Had to address many edge cases in `executeAdd` in the `Parser` class given the variability in user 
+  inputs. This included # of inputs, adding characters instead of integers for
+  rating, or adding a "/" delimiter in user input. 
+* **New Feature**: Remove Command
+  * What it does: Allows users to remove a review for a given type.
+  * Justification: This is an essential behavior to allow users to remove unwanted content.
+  * Highlights: It was difficult to figure out how indexing would work. The list command separates out content based
+  on category (so indexes are seperated per category). However, on the other hand, storage stores all reviews in one
+  array. I had to create a search algorithm to convert between these discrepancies.
+* **New Feature**: List Command
+  * What it does: Displays added reviews to users. Program separates reviews based on category.
+  * Justification: Allows user to see reviews added. Also provides index numbers for removing reviews.
+* **New Feature** Find Command
+  * What it does: Finds a review with a specific title
+  * Justification: Allows user to find a particular review without having to sort through the long list of reviews.
+* **New Feature** Review List
+  * What it does: Stores reviews in current session in array list
+  * Justification: We needed an object to store all the user's reviews and have the appropriate
+  accessors and behavior methods to perform all necessary actions.
+  * Highlights: Follows the OOP model by creating an object for storage. 
+  
+**Timeline:**
```

Comment:
Perhaps the PPP format should be standardised? Timeline info seems to be highlighted above already.